src/main/charts/confluence/templates/ingress.yaml

```diff
@@ -0,0 +1,27 @@
+{{- if .Values.confluence.ingress.create }}
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: {{ include "confluence.fullname" . }}
+  labels:
+    {{- include "confluence.labels" . | nindent 4 }}
+  {{- with .Values.confluence.ingress.annotations }}
+  annotations:
+    {{- toYaml . | nindent 4 }}
+  {{- end }}
+spec:
+  rules:
+    - host: {{ .Values.confluence.ingress.host }}
+      http:
+        paths:
+          - path: "/"
+            backend:
+              serviceName: {{ include "confluence.fullname" . }}
+              servicePort: {{ $.Values.confluence.service.port }}
+          {{ if .Values.synchrony.enabled }}
```

Comment:
The Confluence chart's ingress has an extra rule rulef ro Synchrony (if it's enabled)